presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -207,6 +209,14 @@ public void testAndStats()
                                 .highValue(0.0)
                                 .distinctValuesCount(15.0)
                                 .nullsFraction(0.0));
+
+        // Impossible, with symbol-to-expression comparisons
+        assertExpression(and(
+                new ComparisonExpression(ComparisonExpressionType.EQUAL, new SymbolReference("x"), new ArithmeticBinaryExpression(ADD, new DoubleLiteral("0"), new DoubleLiteral("1"))),
+                new ComparisonExpression(ComparisonExpressionType.EQUAL, new SymbolReference("x"), new ArithmeticBinaryExpression(ADD, new DoubleLiteral("0"), new DoubleLiteral("3")))))
+                .outputRowsCount(0)
+                .symbolStats(new Symbol("x"), SymbolStatsAssertion::emptyRange);
+        // TODO .symbolStats(new Symbol("y"), SymbolStatsAssertion::emptyRange);
```

Comment:
what's up with this todo?

Reply:
This waits for the normalizers. It's already resolved on the CBO epic branch